R/summarise_country.R

```diff
@@ -0,0 +1,163 @@
+summarise_sheet_indicator_list <- function(billion) {
```

Comment:
Even for smaller functions like this, we should try and document them using roxygen even if not exported.

Reply:
I realized after that I didn't use the function later and don't see a use for export, so I removed it.